iib/web/api_v1.py

```diff
@@ -54,6 +54,7 @@ def _get_rm_args(payload, request_id, overwrite_from_index):
         payload.get('add_arches'),
         overwrite_from_index,
         payload.get('overwrite_from_index_token'),
+        payload.get('distribution_scope'),
```

Comment:
I think the API args should be updated

Reply:
Where would I do that? I don't see a place to do this, it appears to all be auto-generated?

Reply:
It's defined in iib/web/static/api_v1.yaml

Reply:
Done